# Geneious-Walk-Through in 5 steps
this is my walkthrough on how to use geneious for my bioinformatics thesis class

# 1st: download simply by going to https://www.geneious.com/ (free trials are okay but often have a lot of necessary restricted features, so reach out to your PI if you need a license) 

# 2nd: learning how files are organized and how to import, download, and export them will be important. 
- files can be **imported** many ways:
1. you can copy and paste them
2. press the add button on the top toolbar and press the new sequence
3. you can import them from your computer, also located in the add section
4. or you can search for your data through NCBI which should be in the left section under "NCBI".
- This allows you to search for genes, genomes, nucleotides, popsets, proteins, PubMed, and so on. 
- all files are organized on the left in folders. Luckily, as a lot of computers store files like this as well, it will be pretty familiar.
- But the key to ensuring you don't mix up your data will be to organize your files well!
- files can be **exported** using the "export" button and can be FASTA or Genbank files

![image](https://github.com/angcarcione/Geneious-Walk-Through/assets/151074947/7d0cdc06-17b6-405b-9df5-f9d2c265560b)

 
# 3rd: how to view documents and sequences.
- To view your documents **select a folder**, this will show all the documents in that folder as well as descriptions for each of the files. In the viewer window at the bottom, you will be able to see the visualization of that file. It can be a protein, a nucleotide sequence, a circular sequence, alignments, and many more. 
- you can **expand** or pop out the document viewer by clicking the 3 buttons in the top right of the viewer window. you can add more panels by pressing split.
- there are tabs in the viewer window "sequence view" with the sequence in visualized format, "annotations" showing notes and features added such as exons and promoters. These may already be present if you downloaded your sequence from NCBI. However, if you are working on an unknown sequence, you will have to annotate it yourself. The "linage" tab will tell you where your data comes from, and "info" will tell you access time, modification, description, and soon. 
- to **zoom** into a sequence you can use the magnifying glass, or you can zoom in with your mouse. this will bring you up until you see the nucleotides in the sequence. you can also highlight to zoom. 
- in the "general" area on the very right you can change things such as nucleotide color, translation, and restriction sites, as well as the complement to the sequence. 
- there is also a graphs tab that can allow you to see GC content
- other tabs will give you more information about restriction analysis, layout, and statistics about the sequence like nucleotide and codon frequencies.
- if you press the shift tab, you can stack all the changes you want to see in the sequence

![image](https://github.com/angcarcione/Geneious-Walk-Through/assets/151074947/f8672181-991f-434c-883a-41b3c5448e23)
![image](https://github.com/angcarcione/Geneious-Walk-Through/assets/151074947/fd4805d7-9ece-4012-b5df-be8a6a1fd507)


# 4th: preforming analyses 
- first, select your file of interest, and then click one of the options of analysis at the **top in the toolbar**. These include **BLAST, WorkFLows, Assemble/align, tree, primers, and cloning**.
- the toolbar is full of shortcuts to many common analyses used in bioinformatics
- Hovering each section in the toolbar will show further analyses you can do in each section. For example, if you hover over cloning, it will give you specific types of cloning, like restriction cloning. 
- Each tool will have specific settings that you can adjust based on what you are researching. For example, in alignment, you can adjust the cost matrix, alignment type, gap open, and gap extension penalties. 
![image](https://github.com/angcarcione/Geneious-Walk-Through/assets/151074947/2b341399-b287-44ef-a11e-b394dbaf0287)

# 5th: annotating sequences
- **annotations** can be very helpful when looking at DNA, RNA, or protein sequences because they give descriptive and visual information about features such as exons, promoters, and genes. 
- the annotation tab is in the document viewer and is a right-pointing arrow. 
- you can filter the types of annotations you want to see. 
- Genbank files probably already include annotations, but there are many ways to make them yourself on your file of interest. 
- you can also import annotations if they are in a file already. 
- to create the annotations you want, highlight them and then click **"add annotation"**. you will also be asked to name it and to pick the type of annotation that it will be. you can also change the direction of your annotation
- you could also **detect or predict annotations** if you don't know of any. this will annotate your file from common features as a reference. you can then increase and decrease the similarity based on how strong you need the similarities to be. 
![image](https://github.com/angcarcione/Geneious-Walk-Through/assets/151074947/358c9cbd-09aa-4f0d-869b-19e05d865399)


# Bonus: what to do if you need a more in-depth tutorial to use geneious yourself
go to: https://www.geneious.com/academy/getting-started/ 
- they have amazing tutorials telling you about every feature you could need to know when using geneious prime. Including how to:
- Sanger sequencing analysis
- Next-generation sequencing analysis
- NGS Pre-Processing, Assembly and Mapping
- Alignments (this is the one I use the most) 
- Phylogenetics
- Cloning
- Primer Design
- CRISPR

_**thanks for reading :) happy researching!**_
